bot/cogs/trivia.py

```diff
@@ -0,0 +1,307 @@
+    )
+    async def trivia(self, ctx: Context, *, category: str = None) -> None:
+        """Trvia command takes in an optional category and sends embed."""
+        if category:
+            category = category.lower()
+        if category == 'categories':
+            embed = Embed(
+                title="Trivia Categories:",
+                description="""
+                        General
+                        Books or Book
+                        Film
+                        Music
+                        Theatre
+                        Televsion or TV
+                        Video Games or VG
+                        Board Games or BG
+                        Science
+                        Computers or Computer
+                        Math
+                        Mythology or Myths
+                        Geography or Geo
+                        History
+                        Art
+                        Animals or Animal
+                        Comics or Comic
+                        Anime
+                        Cartoons or Cartoon
+                            """,
+                color=EMBED_COLOR
+            )
+            embed.set_footer(
+                text="Do .trivia (category name) to choose a category, categories are NOT case sensitive."
+            )
+            await ctx.send(embed=embed)
+            return
+        elif category == 'general':
+            category = 9
+        elif category == 'book' or category == 'books':
+            category = 10
+        elif category == 'film':
+            category = 11
+        elif category == 'music':
+            category = 12
+        elif category == 'theatre':
+            category = 13
+        elif category == 'television' or category == 'tv':
+            category = 14
+        elif category == 'vg' or category == 'video games':
+            category = 15
+        elif category == 'bg' or category == 'board games':
+            category = 16
+        elif category == 'science':
+            category = 17
+        elif category == 'computer' or category == 'computers':
+            category = 18
+        elif category == 'math':
+            category = 19
+        elif category == 'mythology' or category == 'myth':
+            category = 20
+        elif category == 'sports':
+            category = 21
+        elif category == 'geography' or category == 'geo':
+            category = 22
+        elif category == 'history':
+            category = 23
+        elif category == 'art':
+            category = 25
+        elif category == 'animals' or category == 'animal':
+            category = 27
+        elif category == 'comics' or category == 'comic':
+            category = 29
+        elif category == 'anime':
+            category = 31
+        elif category == 'cartoons' or category == 'cartoon':
+            category = 32
+
+        message = await ctx.send(embed=Embed(title='Loading.....', color=EMBED_COLOR))
+
+        while True:
```

Comment:
There should pretty much never be a `while True` in code. This while loop needs a definite end, perhaps after a timer expires, the loop ends. This is a great way to get a cpu leak.